server/app/services/triplers.js

```diff
@@ -103,10 +103,12 @@ async function confirmTripler(triplerId) {
 
     // If this ambassador was once a tripler, then reward the ambassador that
     // initially claimed the then-tripler, only once per upgraded ambassador
-
     let was_once = ambassador.get("was_once");
 
-    if (was_once && !was_once.get("rewarded_previous_claimer")) {
+    // Disable the bonus payouts, at least for now.
+    let ALLOW_BONUS = false;
```

Comment:
can we make this a .env var?

Reply:
Done! I just used the existing `first_reward_payout > 0` which probably makes it easier.